api/client.go

```diff
@@ -0,0 +1,144 @@
+package api
```

Comment:
I'm guessing you'll be missing some coverage here?

Reply:
True, I'll start adding unit tests from this point on since the code base is getting stable-ish.